log/log.go

```diff
@@ -0,0 +1,218 @@
+package log
```

Comment:
Can probably just use https://pkg.go.dev/log/slog for logging, it's pretty similar

Reply:
Nice! I wasn't up to speed on slog essentially replacing logrus. Most of the boilerplate logging,  cobra, and viper was setup using cookiecutter (https://github.com/lacion/cookiecutter-golang). I'll refactor this to use slog, thanks!